src/icons/svg/ipfs.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="1536" height="1792" viewBox="0 0 1536 1792" xmlns="http://www.w3.org/2000/svg">
+    <path d="M 554.26346,256 0,576.00612 V 1215.9939 L 554.26346,1536 1108.5222,1215.9939 V 575.9817 Z m -57.21298,77.07664 h 0.039 c 33.88814,25.00298 80.11555,25.00298 114.0012,0 L 1012.949,565.09766 c -0.8048,7.13568 -0.8031,14.33423 0,21.47017 L 611.06625,818.58886 c -33.88565,-25.00306 -80.10581,-25.00306 -113.99145,0 L 95.192189,586.57268 c 0.803141,-7.13559 0.803159,-14.33908 0,-21.47502 z m 554.48332,317.5842 0.025,0.0104 c 5.7782,4.26366 12.0176,7.86038 18.599,10.73266 v 464.0321 c -38.5956,16.8454 -61.7083,56.8794 -56.9982,98.7277 l -401.88276,232.0159 c -5.77837,-4.2636 -12.0176,-7.8602 -18.59899,-10.7325 l -0.4498,-461.45923 c 38.59548,-16.84529 61.70825,-56.8742 56.99817,-98.72253 z M 56.993277,653.22924 458.8515,885.25512 c -4.71009,41.84833 18.40243,81.88261 56.99816,98.72746 v 464.03222 c -6.58137,2.8722 -12.82088,6.4738 -18.59898,10.7374 L 95.392442,1226.7363 c 4.710078,-41.8483 -18.402431,-81.8828 -56.998157,-98.7276 V 663.97653 c 6.581616,-2.87228 12.815979,-6.47403 18.59409,-10.73759 z" fill="#000"/>
```

Comment:
`svgo` optimized the `fill` attribute away when I built the font locally. Is that supposed to be intended?

The IPFS logo on their website has a filled element in their logo. I am not sure whether it would be in accordance with their brand guidelines to have an unfilled element.

I added a background to the `<body>` and this is how it looked : 

![image](https://user-images.githubusercontent.com/4771718/132123354-f84e96fa-7c85-4aba-a71a-1361bf6a08bc.png)

Reply:
I had to remove the infill because the icon has to be made of a single color.

Reply:
The `fill` element is in the code on this line, but `svgo` _optimizes_ it away for me. Does it not do that for you?

Reply:
I was refering to this

> The IPFS logo on their website has a filled element in their logo. I am not sure whether it would be in accordance with their brand guidelines to have an unfilled element.

Reply:
Don't know where the "fill" attrbute came from. I removed it.